### ISSUE TITLE
Link Nut in JsonTestSuite

### ISSUE THREAD

**Issue by FauconFan** [Heijix/nut-lang#12]
Add nut parsers to this project:

https://github.com/nst/JSONTestSuite

**Comment by xavitator** [comment]
I think we can close this issue.

**Comment by FauconFan** [comment]
No, i want to add the nut parsers to all parsers in the JSON test suite...

**Comment by xavitator** [comment]
haaa ok, i understand yet what is precisely JsonTestSuite, sorry x).
